py/pythonhelpers.h

```diff
@@ -688,6 +704,10 @@ class PyDictPtr : public PyObjectPtr {
 
 };
 
+/* TODO - is there a better way than this?? */
+#ifndef PyMethod_GET_CLASS
+#define PyMethod_GET_CLASS(m) (PyObject*)Py_TYPE(PyMethod_GET_SELF(m))
```

Comment:
The function has been removed in Python 3. Technically, the method which uses it isn't used anywhere in the wrapper. This and the relevant method in `PyMethodPtr` can be removed, if you like.